.github/workflows/tests.yml

```diff
@@ -166,6 +166,19 @@ jobs:
       - uses: ./.github/actions/setup-rust
       - run: cargo test
 
+  test_features:
+    strategy:
+      matrix:
+        features: [ "serde", "rdf-star", "serde rdf-star" ]
+        os: [ "linux", "windows", "macos" ]
+    runs-on: ${{ matrix.os }}-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      - uses: ./.github/actions/setup-rust
+      - run: cargo test --features ${{ matrix.features }}
```

Comment:
This sounds a bit overkill. I believe the current tests already cover what we care about

Reply:
I've removed the multiple os' and cut out the `rdf-star` combos. I would like to make sure that both `serde` and `serde-unvalidated` are tested.